views/components/Certification.js

```diff
@@ -1,72 +1,303 @@
-          </li>
-          <li className="process__step-info process__step-info_deploy">
-            <h3>
-              <spana>3</spana>
-              &mdash;Second place
-            </h3>
-            <p>
-              Got second prize for the competitive algorithm contest of the
-              university
-            </p>
-          </li>
-          <li className="process__step process__step_deliver">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deliver">
-            <h3>
-              <spana>4</spana>
-              &mdash;First place
-            </h3>
-            <p>
-              Highschool, got First prize for the competitive algorithm contest
-              of the city.
-            </p>
-          </li>
-        </ul>
+import React, { useState, Fragment } from 'react'
+import styled from 'styled-components/macro'
+import CertIcon from '~/views/components/common/CertIcon'
+import { CERTIFICATIONS } from '~/consts/pages'
+import { CERT_ACTIVE_ITEM } from '~/consts/certification'
+import PropTypes from 'prop-types'
+import Container from '~/views/components/common/Container'
+import Header from '~/views/components/common/Header'
+import { Translate } from 'react-localize-redux'
+
+const Certification = () => {
+  const [itemSelected, setSelectedItem] = useState(CERT_ACTIVE_ITEM[0])
+
+  return (
+    <Container id={CERTIFICATIONS}>
+      <div className="container">
+        <Header>
+          <h2>Certifications and Prizes</h2>
+        </Header>
+        <List className="certification">
+          <Research
+            itemSelected={itemSelected}
+            setSelectedItem={setSelectedItem}
+            title="CertResearch"
+            content="CertContentResearch"
+          />
+          <Develop
+            itemSelected={itemSelected}
+            setSelectedItem={setSelectedItem}
+            title="CertDevelop"
+            content="CertContentDevelop"
+          />
+          <Deploy
+            itemSelected={itemSelected}
+            setSelectedItem={setSelectedItem}
+            title="CertDeploy"
+            content="CertContentDeploy"
+          />
+          <Deliver
+            itemSelected={itemSelected}
+            setSelectedItem={setSelectedItem}
+            title="CertDeliver"
+            content="CertContentDeliver"
+          />
+        </List>
       </div>
-    </div>
-  </section>
-)
+    </Container>
+  )
+}
 
 export default Certification
+
+const Research = ({ itemSelected, setSelectedItem, title, content }) => {
+  return (
+    <Fragment>
+      <ResearchItem onClick={() => setSelectedItem(CERT_ACTIVE_ITEM[0])}>
+        <CertIcon
+          id={CERT_ACTIVE_ITEM[0]}
+          itemSelected={itemSelected}
+          name="c"
+        />
+      </ResearchItem>
+
+      <ResearchInfo itemSelected={itemSelected}>
+        <Title>
+          <span>1</span> &mdash; <Translate id={title} />
```

Comment:
use 1 is cool, but can we use CERT_ACTIVE_ITEM[0] for this?  🤔

Reply:
Updated